### ISSUE TITLE
reference from the function twl_probe() to the function .init.text:unprotect_pm_master

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#318]
Forked from #6 

```
WARNING: vmlinux.o(.text+0x3f03ed5): Section mismatch in reference from the function twl_probe() to the function .init.text:unprotect_pm_master()
The function twl_probe() references
the function __init unprotect_pm_master().
This is often because twl_probe lacks a __init 
annotation or the annotation of unprotect_pm_master is wrong.
```
related to asm goto or just a simple section mismatch bug?

**Comment by nathanchance** [comment]
Hmmm is your tree up to date? This should have been fixed already, as this is a dupe of #222. Feel free to close it if not.

**Comment by nickdesaulniers** [comment]
my tree is not up to date.  Sorry for the dupes!
